src/Symfony/Bridge/Doctrine/Attribute/MapEntity.php

```diff
@@ -21,13 +21,49 @@
 class MapEntity extends ValueResolver
 {
     public function __construct(
+        /**
+         * The entity class.
+         */
         public ?string $class = null,
+
+        /**
+         * Specify the object manager used to retrieve the entity.
+         */
         public ?string $objectManager = null,
+
+        /**
+         * An expression to fetch the entity. The following variables are available:
```

Comment:
this should mention that this is an expression using the ExpressionLanguage syntax, as done in the other PR.